src/page/Changelog.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import './Changelog.css';
+
+class Changelog extends React.Component {
+
+    getWeenieThumbnail()
+    {
+        return (
+            <svg className="bi bi-person-fill" width="2em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                <path fillRule="evenodd" d="M3 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1H3zm5-6a3 3 0 100-6 3 3 0 000 6z" clipRule="evenodd"/>
+            </svg>
```

Comment:
[MINOR] This is a bit hard to read. I would suggest we put it in a constant or extract it in a component of it's own.